py_gtfs_rt_ingestion/ingest.py

```diff
@@ -74,35 +82,25 @@ def lambda_handler(event: dict, context) -> None:
     {
         files: [file_name_1, file_name_2, ...],
     }
-
-    or
-    {
-        s3_files: [file_name_1, file_name_2, ...],
-        bucket: bucket_name
-    }
+    S3 files will begin with 's3://'
 
     batch files should all be of same ConfigType
     """
     logging.info("Processing event:\n%s" % json.dumps(event, indent=2))
 
     # get files and function to read them based on the event. for local files,
     # use gzip reading, for s3 files use pyarrow to read directly from s3
-    if 'files' in event:
-        files = event['files']
-        conversion_func = gz_to_pyarrow
-    elif 's3_files' in event:
-        files = event['s3_files']
-        conversion_func = s3_to_pyarrow
-    else:
+    if 'files' not in event:
         raise ArgumentException(
-            "Unable to find 'files' or 's3_files' in event json")
+            "Unable to find 'files' in event json")
+    files = event['files']
```

Comment:
Use 'files' key for both event types. Perform s3 check in `convert.py` `gz_to_pyarrow` function